improver_tests/calibration/test_init.py

```diff
@@ -542,6 +506,72 @@ def test_duplicate_forecasts(self):
             )
 
 
+@pytest.mark.parametrize("probability_forecasts", (True, False))
+@pytest.mark.parametrize("include_land_sea_mask", (True, False))
+@pytest.mark.parametrize("include_additional_predictor", (True, False))
```

Comment:
Given that the use of parameterisation has condensed these tests a lot, it might be useful to add an additional test here for either 0, 1 or 2 additional predictors, just to check that multiple additional predictors are handled i.e.

```
@pytest.mark.parametrize("number_of_additional_predictors", (0, 1, 2))
```

Reply:
Changed `include_additional_predictor ` taking values `True` and `False` to `n_additional_predictors` taking values in `(0,1  ,2 )`